src/test/resources/com/dnsimple/cancelDomainTransfer/success.http

```diff
@@ -0,0 +1,30 @@
+HTTP/1.1 202 Accepted
+Server: nginx
+Date: Wed, 29 Apr 2020 19:49:41 GMT
+Content-Type: application/json; charset=utf-8
+Transfer-Encoding: chunked
+Connection: keep-alive
+Status: 202 Accepted
+X-RateLimit-Limit: 2400
+X-RateLimit-Remaining: 2397
+X-RateLimit-Reset: 1588193270
+ETag: W/"8404bf2739e44e9f69f3a5199466776d"
+Cache-Control: no-store, must-revalidate, private, max-age=0
+X-Request-Id: 10d75bcf-0066-4d38-98a2-dc18dd5b6f4c
+X-Runtime: 1.752154
+Strict-Transport-Security: max-age=31536000
+
+{
```

Comment:
This is weird. Does our API pretty format JSON documents? I've checked that this is what we have at https://github.com/dnsimple/dnsimple-developer/blob/master/fixtures/v2/api/cancelDomainTransfer/success.http but it feels weird anyway.

Reply:
goot point @ggalmazor . How did you generate those files @duduribeiro ? Did you edit them manually?

Reply:
@ggalmazor @weppos it is how my rest client plugin returned the data. It returned formatted for me.

Reply:
We use `curl -i -s ...`. I suppose we'll have to document it anyways, actually create a shell script to make sure we all use the same approach, as per conversation with @ggalmazor related to the current issue of transfer chunked.